src/sass/disability-benefits.scss

```diff
@@ -9,3 +9,11 @@
 // @import "modules/m-omb-info";
 @import "modules/m-form-confirmation";
 
+
+.dashed {
```

Comment:
What do you think about doing `li.dashed`? (Since--if I understand this correctly--this is used to replace the `<li>`'s bullet with a `-`.)

Reply:
I tend to avoid adding specificity so that styles can be more easily overridden later. If another `dashed` class needs to be created later on, it would make sense to either rename this to be more specific itself or add specificity as you propose.

Reply:
If you think a more descriptive name makes sense here, let's go for it.

Reply:
How do you like `.dashed-bullet`? Seems a little on the nose, but gets the job done. I'm not strongly opinionated, though.